client/src/components/profile/editprofile.js

```diff
@@ -69,6 +80,57 @@ class EditProfilePopup extends Component {
       country: e.target.value
     })
   }
+  
```

Comment:
These boilerplate methods should be refactored into a separate `editavatar.js` file.